### PR TITLE
Update composer dependencies

### PR DESCRIPTION
**Pull request by dominikvoda**

### CODE REVIEW

**Review by tomasJancar — APPROVED** [review]

File: src-php/Grid/GridColumn/GridColumnOption.php [hunk 1]
```diff
@@ -21,7 +21,6 @@ final class GridColumnOption extends Enum
     public const XS_10 = 'col-xs-10';
     public const XS_11 = 'col-xs-11';
     public const XS_12 = 'col-xs-12';
-
```
Comment (michal-tichy-84) [inline src-php/Grid/GridColumn/GridColumnOption.php:24]: [O] I would keep these lines

File: src-php/Grid/GridRowOption.php [hunk 2]
```diff
@@ -12,23 +12,18 @@ final class GridRowOption extends Enum
     public const XS_START = 'start-xs';
     public const XS_CENTER = 'center-xs';
     public const XS_END = 'end-xs';
-
```
Comment (michal-tichy-84) [inline src-php/Grid/GridRowOption.php:15]: [O] I would keep these lines

File: src-php/Table/Model/ArrayDataProvider.php [hunk 3]
```diff
@@ -30,6 +30,9 @@ public function __construct(array $data)
     }
 
 
+    /**
+     * @return TableIterator<RowData>
```
Comment (michal-tichy-84) [inline src-php/Table/Model/ArrayDataProvider.php:34]: :+1:

File: src-php/Table/Ui/Table.php [hunk 4]
```diff
@@ -141,11 +137,6 @@ private function renderBody(): string
     }
 
 
-    /**
-     * @param string $key
-     *
-     * @return callable(CellData $cellData, RowData $rowData, ColumnDefinition $columnDefinition, TableIterator $iterator): Cell
```
Comment (michal-tichy-84) [inline src-php/Table/Ui/Table.php:147]: [O] The description of callable might stay here

**Review by michal-tichy-84 — APPROVED** [review]

**Pull request merged**